operator/webhooks/pod_mutator/pod_mutating_webhook.go

```diff
@@ -184,8 +185,13 @@ func (a *PodMutatingWebhook) copyAnnotationsIfParentAnnotated(ctx context.Contex
 			return false, nil
 		}
 
+		if rsOwner.Kind == "Rollout" {
+			ro := argov1alpha1.Rollout{}
+			return a.fetchParentObjectAndCopyLabels(ctx, podOwner.Name, req.Namespace, pod, &ro)
```

Comment:
just a nitpick, but just to stay consistent within this function:

```suggestion
			ro := &argov1alpha1.Rollout{}
			return a.fetchParentObjectAndCopyLabels(ctx, podOwner.Name, req.Namespace, pod, ro)
```